README.md

```diff
@@ -148,6 +148,19 @@ GC_OPTS="${GC_OPTS}
          ${YOUR_EXTRA_GC_OPTIONS}"           
 ```
 
+## Custom contexts
+
+Spark-Job-REST supports custom job context factories defined in `context.job-context-factory`.
```

Comment:
I would add "property" at the end for better understanding. Also I think that it's worth mentioning when to set this config (eg. at context creation)